<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformProjDialog</class>
 <widget class="QDialog" name="TransformProjDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transform projection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/plugins/CartoGrapher/resources/m.svg</normaloff>:/plugins/CartoGrapher/resources/m.svg</iconset>
  </property>
  <widget class="QgsProjectionSelectionWidget" name="Proj_selectBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="layerCbmBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>160</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input layer or shp：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Projecion transformed：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="output_checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>351</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Open output file after running algorithm</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QgsFileWidget" name="output_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
